docs/versioned_docs/version-3.5/manual/running.md

```diff
@@ -340,6 +340,8 @@ In the bottom left corner of the screen, look for <span class="menuItems">Prefer
 |Display of the reconciliation preview on hover|cell-ui.previewMatchedCells|Boolean|true|false|v3.2|
 |Values for the choice of the number of rows to display|ui.browsing.pageSize|Array of number (JSON)|[ 5, 10, 25, 50 ]|[ 100, 500, 1000 ]|v3.4|
 |Width of the panel for facets/history|ui.browsing.facetsHistoryPanelWidth|Number (pixel)|300|500|v3.5|
+|Keep the facet's splash hidden|ui.browsing.hideFacetSplash|Boolean|false|true|—|
+|Keep the history's splash hidden|ui.history.hideHistorySplash|Boolean|false|true|—|
```

Comment:
```suggestion
|Keep the facet's splash hidden|ui.browsing.hideFacetSplash|Boolean|false|true|—|
|Keep the history's splash hidden|ui.history.hideHistorySplash|Boolean|false|true|—|
```

I've just seen this: `ui.browsing` must stay consistent. I understand why you went for that (other naming in the code structure), but at the user level, all of this is browsing, one is about the facet panel and the other about the history panel, both of which are in the `leftPanel`.

So, the logic of name is: `ui.browsing.hide{panelName}Splash`, where `panelName` = `Facet` of `History`.

Regards, Antoine

Reply:
@antoine2711: Okay